pkg/cmd/target/history.go

```diff
@@ -0,0 +1,131 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package target
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/gardener/gardenctl-v2/internal/util"
+	"github.com/gardener/gardenctl-v2/pkg/cmd/base"
+)
+
+const (
+	historyFile string = "history"
+)
+
+// NewCmdHistory returns a new target history command.
+func NewCmdHistory(f util.Factory, o base.Options) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "history",
+		Short: "Print the target history",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runHistoryCommand(f, o)
```

Comment:
please follow the pattern as we do it for the other commands (complete, validate, run). Except there is a strong reason not to do so. We also have a convenience function for this purpose https://github.com/gardener/gardenctl-v2/blob/master/pkg/cmd/env/env.go#L26